### DISCUSSION TITLE
Help me with using this library with platform IO + esp32s3

### DISCUSSION THREAD

**Discussion by Trishanu123** [pschatzmann/arduino-espeak-ng#17] [Q&A] [answered]
Hello, I need some help with using this library with the esp32s3. I would appreciate it if you can come on a call with me. My email is gtrishanu@gmail.com.

**Comment by Trishanu123** [comment]
want some help with the set up of the library and platform io.

**Comment by pschatzmann** [comment, marked as answer]
Just add it an external Git Resource as described in the [PlatformIO documentation](https://docs.platformio.org/en/latest/projectconf/sections/env/options/library/lib_deps.html)

**Comment by Trishanu123** [comment]
If you dont mind, can we please catch up on a gmeet and figure things out? I will highly appreciate it if you can contact me through gtrishanu@gmail.com

**Comment by pschatzmann** [comment]
I don't see any reason to do this. 
Feel free to open a discussion if you think that the documentation is not complete.

I don't understand why you even created this discussion in the first place. In the Wiki there is a chapter about PlatformIO!
